huaweicloud/services/acceptance/dns/resource_huaweicloud_dns_zone_v2_test.go

```diff
@@ -264,5 +244,5 @@ resource "huaweicloud_dns_zone" "zone_1" {
     owner     = "terraform"
   }
 }
-	`, zoneName, HW_ENTERPRISE_PROJECT_ID_TEST)
+	`, zoneName, acceptance.HW_ENTERPRISE_PROJECT_ID_TEST)
```

Comment:
```suggestion
`, zoneName, acceptance.HW_ENTERPRISE_PROJECT_ID_TEST)
```

please check the format in all testing files of DNS.

Reply:
OK. I have checked all testing files.